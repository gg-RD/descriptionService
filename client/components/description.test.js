import React from 'react';
import { shallow, mount, render } from 'enzyme';
import Description from './description.jsx'


describe('Description Component', () => {
	it('Description Component should render as expected', () => {
		const wrapper = shallow(<Description />)
		expect(wrapper).toMatchSnapshot();
	});

});

