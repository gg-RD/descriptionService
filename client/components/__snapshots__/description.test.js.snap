// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Description Component Description Component should render as expected 1`] = `
<div
  className="container"
>
  <div
    className="blank"
  >
    something else
  </div>
  <div
    className="content"
  >
    <ShoeName
      data={
        Array [
          Object {
            "shoe_cat": "",
            "shoe_descpription": "",
            "shoe_name": "",
            "shoe_price": "",
            "shoe_shown": "",
            "shoe_size": Array [],
            "shoe_style": "",
          },
        ]
      }
    />
    <SizeTable />
    <AddToCart />
    <div
      className="shippings-main"
      onClick={[Function]}
    >
      FREE SHIPPING & RETURNS
      <div
        className="icon"
      >
        <FontAwesome
          name="fas fa-plus"
        />
         
      </div>
    </div>
    <div
      className="hidden"
    >
      <ShippingReturns />
    </div>
    <div
      className="shippings-main"
      onClick={[Function]}
    >
      REVIEWS (3)
      <div
        className="icon"
      >
        <FontAwesome
          name="fas fa-plus"
        />
         
      </div>
    </div>
    <div
      className="hidden"
    >
      <Reviews />
    </div>
    <ShoeDescription
      data={
        Array [
          Object {
            "shoe_cat": "",
            "shoe_descpription": "",
            "shoe_name": "",
            "shoe_price": "",
            "shoe_shown": "",
            "shoe_size": Array [],
            "shoe_style": "",
          },
        ]
      }
    />
  </div>
</div>
`;
